unbound/manifest.json

```diff
@@ -0,0 +1,11 @@
+{
```

Comment:
Needs a "guid" entry so that it can build a windows package (since it says supported_os includes windows).  See https://github.com/DataDog/integrations-extras/blob/master/snmpwalk/manifest.json#L8 for an example.  LMK if you need help generating a guid.